BALSAMIC/utils/constants.py

```diff
@@ -212,10 +212,25 @@
         "field": "FORMAT"
     },
     "pop_freq": {
-        "tag_value": 0.005,
+        "tag_value": 0.001,
         "filter_name": "balsamic_high_pop_freq",
         "field": "INFO"
     },
+    "qss": {
+	"tag_value": 20 ,
+	"filter_name": "balsamic_low_quality_scores",
+	"field": "FORMAT"
+    },
+    "cnt_reads": {
```

Comment:
```suggestion
    "strand_reads": {
```

To make it obvious that this is aimed for strand specific counts.